recipe/meta.yaml

```diff
@@ -44,14 +44,13 @@ requirements:
     - python
```

Comment:
doesn't it need ``pybind11``

Reply:
https://github.com/google/pytype/blob/07662febd183e4a3395e05ea8d3adf75397bd7cb/pyproject.toml#L2

Reply:
I wonder how this is passing without it: https://github.com/google/pytype/blob/07662febd183e4a3395e05ea8d3adf75397bd7cb/setup.py#L18-L24

Reply:
It's pulling in a pybind11 folder in line 16? I didn't look into the extra files.
Looking at them now - those are not actually patch sets but are pulling in some specific commit revisions of googletest, pybind11, and typeshed. These probably should be build-/test-time dependencies.

Reply:
Ah! https://github.com/Anthchirp/pytype-feedstock/blob/f811a2dd93459a3cf28ea5fdb853814388ca2c19/recipe/meta.yaml#L13-L21

Reply:
Would you wanna unvendor these or nah?

Reply:
There is no real need --- the pkg doesn't end up taking these deps with it; they're just needed at build time, I believe.

Reply:
I think it would be neater to unvendor, but I feel that's a separate endeavour